FIDO2_get_next_credential_req.md

```diff
@@ -0,0 +1,16 @@
+## FIDO2 Get Next Credential Request
```

Comment:
this file isn't referenced by anything and seems to be obsolete from what I see.

Reply:
Yeah, forgot to remove all these deprecated files. Will fix.